docs/configuration.md

```diff
@@ -76,6 +76,90 @@ systems.
 Section for Worker configurations that should only be applied to Windows-based
 systems.
 
+### status
+
+Watchmaker supports posting the watchmaker status to status providers. Watchmaker status values are one of: 'Running', 'Failed', or 'Completed'. Each status provider defines what it means to "post the status".. Currently, the supported provider types include: 'aws' and 'azure'. These status providers both post the status as a tag to the instance/VM.
+
+Providers have the ability to detect whether the system is compatible with the provider type. In order to post status, the system running watchmaker must be compatible with the status provider type. For example, the 'azure' provider will be skipped when watchmaker is running on an AWS EC2 instance, and vice versa.
+
+Prerequisites for AWS:
```

Comment:
This doc section was reworked in https://github.com/plus3it/watchmaker/pull/2585. I'd suggest skimming that changeset for the diff and updating this to match.